client/src/app/site/agenda/components/list-of-speakers/list-of-speakers.component.html

```diff
@@ -91,7 +91,12 @@ <h1 class="los-title">
 
     <!-- Waiting speakers -->
     <div class="waiting-list" *ngIf="speakers && speakers.length > 0">
-        <os-sorting-list [input]="speakers" [live]="!isSortMode" [count]="true" [enable]="opCanManage() && isSortMode">
+        <os-sorting-list
+            [input]="speakers"
+            [count]="true"
+            [enable]="opCanManage() && isSortMode"
+            (sortEvent)="onSortingChanged($event)"
```

Comment:
without `[live]="true"` other clients will not be informed about changes. That is required. everything else looks good.

- open two clients on the same list of speakers
- sort something
- save
- changes will currently not reflect